com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/FlowableFacade.java

```diff
@@ -254,6 +254,11 @@ public void trigger(String userId, String executionId) {
         }
     }
 
+    public void setProcessVariable(String executionId, String varName, Object varValue) {
```

Comment:
Make it consistent, rename it to setVariable.
Process is very specific, you're working with Executions which is abstract.